packages/eslint-config/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@agoric/eslint-config",
-  "version": "0.1.0+1-dev",
+  "version": "0.1.1",
```

Comment:
This will be a first publish. Version should be `0.1.0`.

Reply:
Should also rename `@endo/eslint-config` and sync with `@agoric/eslint-config` before publish (so we can scuttle the latter)

Reply:
Again, you've consumed `0.1.0`.